lib/checks/tables/td-headers-attr.js

```diff
@@ -43,20 +39,22 @@ var badCells = cells.reduce(function(badCells, cell) {
 		}
 
 		// Check if the headers are of cells inside the table
-		notOfTable = headers.reduce(function(fail, header) {
-			return fail || ids.indexOf(header) === -1;
-		}, false);
+		notOfTable = headers.find(header => !ids.includes(header));
 
 		if (isSelf || notOfTable) {
 			badCells.push(cell);
 		}
 	}
-	return badCells;
-}, []);
+});
+
+if (reviewCells.length) {
```

Comment:
If there are bad cells, this check needs to return false, not undefined. Best to move this if down.